### PR TITLE
updates changelog in prep for releasing5.6rc

### PR DESCRIPTION
**Pull request by mpacer**
This is a PR to update the changelog before the 5.6 release candidate.

Once we merge this unless someone else specifically objects I'm going to move forward with getting 5.6rc1 out the door.

@takluyver @Carreau @minrk

### CODE REVIEW

**Comment by minrk** [comment]
Looks good! We should also make sure to resolve #3668 before we release 5.6.

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -20,6 +20,40 @@ We strongly recommend that you upgrade to version 9+ of pip before upgrading ``n
...
+
+Fixing problems:
+
+- Don't duplicate token in displayed URL (:ghpull:`3656`)
+- Don't trash non-empty directories on Windows (:ghpull:`3673`)
+- Include LICENSE file in wheels (:ghpull:`3671`)
+
+Testing: 
+- Add find replace test (:ghpull:`3630`)
+- [3335] Convert JS tests to Selenium (:ghpull:`3601`)
```
Comment (takluyver) [inline docs/source/changelog.rst]: Let's get rid of the `[3335]`, and preferably be more specific which JS tests were converted. Unfortunately PR titles don't all make good release notes.

File: docs/source/changelog.rst [hunk 2]
```diff
@@ -20,6 +20,40 @@ We strongly recommend that you upgrade to version 9+ of pip before upgrading ``n
...
     ``pip --version``.
+    
+.. _release-5.6.0:
+
+5.6.0
+-----
+
+New features:
+
+- Execute cells by clicking area in input prompt (:ghpull:`3535`, :ghpull:`3687`)
```
Comment (takluyver) [inline docs/source/changelog.rst]: This got changed from an invisible area to an icon

File: docs/source/changelog.rst [hunk 3]
```diff
@@ -20,6 +20,40 @@ We strongly recommend that you upgrade to version 9+ of pip before upgrading ``n
...
+- Update to jQuery 3.3 (:ghpull:`3655`)
+
+Fixing problems:
+
+- Don't duplicate token in displayed URL (:ghpull:`3656`)
+- Don't trash non-empty directories on Windows (:ghpull:`3673`)
+- Include LICENSE file in wheels (:ghpull:`3671`)
+
+Testing: 
+- Add find replace test (:ghpull:`3630`)
```
Comment (takluyver) [inline docs/source/changelog.rst]: You may need a blank line above this, I'm not sure.

**Comment by mpacer** [comment]
Ok then I'll move forward with the release @takluyver thanks for the help & merge!

**Pull request merged**